### PR TITLE
52 complete styling for summarydetail component amt

### PR DESCRIPTION
**Pull request by allison-truhlar**
This PR links to Issue #52 

### What changes did you make?
-  The close button in SummaryDetail now stays fixed in the upper right corner - I added "fixed" to the Tailwind classes in the button element.
- SummaryDetail now "slides" in and out. To achieve this, I removed the conditional rendering of SummaryDetail in App.jsx. SummaryDetail is now always rendered, and whether it is viewable or not is controlled by the CSS.
- The picture at the top of SummaryDetail spreads across the entire width of the component and zoom in on the image, without increasing the height of the div it's contained in. I followed the example of the CSS in the RecipeCard components to do this.
- I added bullet points to the ingredient list.
- I made the Nutrition element a flex box so the description of the nutrition element and the value can be justified with space between. I also bolded the calories row of the Nutrition element.
- I added media queries for the container around the Ingredients element and Nutrition element, to change the flex direction from column on cellphones to row on all other screen sizes.

### Why did you make these changes?
- To more closely match the Figma design specifications.

### Screenshots

<details><summary>Summary detail top image and close button</summary>

Screenshot 2023-11-15 at 10 33 58 AM

</details> 

<details><summary>Cell phone - ingredients and nutrition</summary>

Screen Shot 2023-11-15 at 10 29 41

</details> 

<details><summary>Larger screens - ingredients and nutrition</summary>

Screenshot 2023-11-15 at 10 29 55 AM

</details>

### CODE REVIEW

**Comment by Emimint** [comment]
@allison-truhlar I played around with it for a while and I could not see any glitch. It looks great ! Just a small thing: in the Figma design here : 

image

The corners of the summary page appeared rounded, with I think is a nice effect to have. It would be cool to have it here as well, but that is just me.

File: src/components/summaryDetail/SummaryDetail.jsx [hunk 1]
```diff
@@ -12,76 +15,100 @@ export default function SummaryDetail({
...
-  return (
-    <div className={`${isDetailShown ? 'show-summary-detail' : 'hide-summary-detail'} fixed top-0 right-0 h-full max-w-2xl bg-white max-h-screen overflow-y-scroll inline-flex flex-col items-center pb-4`}>
+  const closeBtnStyles = {
+    color: 'white', zIndex: 11, position: 'fixed', top: '1 rem', right: '1 rem',
+  };
 
-      <button onClick={handleSummaryDetailClose} className="self-end fixed top-4 w-4 h-4 bg-gray-700 rounded-full mr-4 p-4 text-white inline-flex items-center justify-center" type="button">X</button>
+  const containerStyle = {
+    transform: isDetailShown ? 'translateX(0)' : 'translateX(100%)',
+    transition: 'transform 1s ease',
```
Comment (AOA19) [inline src/components/summaryDetail/SummaryDetail.jsx]: @allison-truhlar The summary detail component looks really good and it's working great! I just have one small suggestion, the speed at which the component slides in and out is a little slow. I think maybe changing the speed from `1s` to `0.6s` works well.

**Comment by allison-truhlar** [comment]
@Emimint Good catch! I just added the rounded corners for larger screen sizes (the component is full screen at smaller sizes, so it's not necessary). Here is how it looks:

Screenshot 2023-11-15 at 1 28 32 PM

File: src/components/summaryDetail/SummaryDetail.jsx (hunk 1 above)
Comment (allison-truhlar) [reply src/components/summaryDetail/SummaryDetail.jsx]: @AOA19 I can see what you mean - especially when it has to go across the full screen width. I made the change to 0.6s in the latest commit on this branch: [078fb10](https://github.com/chingu-voyages/v46-tier2-team-20/pull/67/commits/078fb1040209bba390ff3561abf2207734c5839b)

**Comment by AOA19** [comment]
@allison-truhlar The speed of the component looks good and so do the rounded corners. I'll go ahead and approve and merge these changes.

**Review by AOA19 — APPROVED** [review]
@allison-truhlar The speed of the component looks good and so do the rounded corners. I'll go ahead and approve and merge these changes.

**Pull request merged**